packages/@o3r/rules-engine/src/public_api.ts

```diff
@@ -2,7 +2,7 @@ export * from './components/index';
 export * from './devkit/index';
 export * from './engine/index';
 export * from './fact/index';
-export * from './facts/index';
+export * from './inner-facts/index';
```

Comment:
So you do not export the `currentTimeSubject$` const anymore, it sounds like a breaking change.
I think you should still export it and flagged it as deprecated until v12